src/SFA.DAS.AssessorService.Web.UnitTests/Helpers/HtmlHelperExtensionsTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using NUnit.Framework;
+using SFA.DAS.AssessorService.Web.Extensions;
+
+namespace SFA.DAS.AssessorService.Web.UnitTests.Helpers
+{
+    [TestFixture]
+    public class HtmlHelperExtensionsTests
+    {        
+        private string expectedLabelOne;
+        private string expectedLabelTwo;
+        private string expectedOutput;
+        private string actualOutput;      
+
+        [Test]
+        public void WhenICallSetZenDeskLabelsWithMultipleLabelsWithMultipleWordsAndApostrophes_ThenTheOutputIsCorrect()
```

Comment:
As the tests are identical, you could have used [TestCases](https://nunit.org/docs/2.5/testCase.html)

Reply:
Added TestCaseSource